testnets/tucanatestnet/chain.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "tucanatestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Tucana Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "tucana_712-1",
+  "bech32_prefix": "tuc",
+  "daemon_name": "tucanad",
+  "node_home": "$HOME/.tucanad",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "atuc",
+        "fixed_min_gas_price": 5000000000,
+        "low_gas_price": 10000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 30000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "atuc"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/TucanaProtocol/Tucana",
+    "recommended_version": "v8.1.0",
+    "compatible_versions": [
+      "v8.1.0"
+    ],
+    "cosmos_sdk_version": "v0.50.8",
```

Comment:
this cosmos sdk version number at this higher level disagrees with the number given in the versions array. This should match the latest version in the versions array.
Same issue with ibc go version

Reply:
Thank you for pointing that out, fixed. Pls review again thank you ! @JeremyParish69